signer/pgp/README.rst

```diff
@@ -0,0 +1,61 @@
+PGP Signing
+===========
+
+.. sectnum::
+.. contents:: Table of Contents
+
+This signer implements the Pretty Good Privacy signature format. It accepts data
```

Comment:
There are multiple versions of this right? Might be good to mention the equivalent gpg command used in the tests